pkg/security/tests/chmod_test.go

```diff
@@ -36,19 +36,27 @@ func TestChmod(t *testing.T) {
 	}
 	defer os.Remove(testFile)
 
-	t.Run("chmod", func(t *testing.T) {
-		if _, _, errno := syscall.Syscall(syscall.SYS_CHMOD, uintptr(testFilePtr), uintptr(0o707), 0); errno != 0 {
+	t.Run("fchmod", func(t *testing.T) {
+		f, err := os.Open(testFile)
```

Comment:
It would be nice to create dedicated rule/file for each test so that we can run them independently, what do you think ?